contrib/http_servlet/src/main/java/io/opentelemetry/contrib/http/servlet/OtelHttpServletUtils.java

```diff
@@ -0,0 +1,78 @@
+/*
+ * Copyright 2019, OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.opentelemetry.contrib.http.servlet;
+
+import static com.google.common.base.Preconditions.checkNotNull;
+import static com.google.common.base.Strings.isNullOrEmpty;
+import static io.opentelemetry.contrib.http.core.HttpTraceConstants.INSTRUMENTATION_LIB_ID;
+
+import com.google.errorprone.annotations.MustBeClosed;
+import io.opentelemetry.OpenTelemetry;
+import io.opentelemetry.context.Scope;
+import io.opentelemetry.contrib.http.core.HttpRequestContext;
+import io.opentelemetry.contrib.http.core.HttpServerHandler;
+import io.opentelemetry.trace.DefaultSpan;
+import javax.servlet.ServletRequest;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+/** Provides static utility methods for working with Java Servlet objects. */
+public class OtelHttpServletUtils {
```

Comment:
I feel like having the `Otel` prefix is everywhere unnecessary. This is all OpenTelemetry code. :)